### PR TITLE
Tidy up the status monitor

### PR DESCRIPTION
**Pull request by Tkael**
Simplify and streamline the code for StatusMonitor.cs. Hopefully improving performance a bit.

### CODE REVIEW

**Comment by richardbuckle** [comment]
From a risk management point of view, since this has no test coverage, I think it's best to hold this one for after the 3.0-rc1 release.

File: StatusMonitor/StatusMonitor.cs [hunk 1]
```diff
@@ -174,128 +168,103 @@ public List<Event> ParseStatusEntry(string line)
...
-                                status.latitude = JsonParsing.getOptionalDecimal(data, "Latitude");
-                                status.longitude = JsonParsing.getOptionalDecimal(data, "Longitude");
-                                status.altitude = JsonParsing.getOptionalDecimal(data, "Altitude");
-                                status.heading = JsonParsing.getOptionalDecimal(data, "Heading");
-
-                                // Spin off a thread to pass status entry updates in the background
-                                Thread updateThread = new Thread(() => handleStatus(timestamp, status));
-                                updateThread.IsBackground = true;
-                                updateThread.Start();
+                                status.gui_focus = "none";
```
Comment (richardbuckle) [inline StatusMonitor/StatusMonitor.cs]: I wonder if an enum would be better than a string for this property.

Comment (Tkael) [reply StatusMonitor/StatusMonitor.cs]: We want the output to be easy for the end user to parse. I think an enum might be a little less user friendly?

Comment (richardbuckle) [reply StatusMonitor/StatusMonitor.cs]: I’ve been spoiled by Swift’s enums, which can be backed by string as easily as by int. Oh well.

**Pull request merged**